### DISCUSSION TITLE
Issue with Selenium: NoSuchDriverException for Firefox on arm64v8/debian:bullseye-slim with Geckodriver 0.34.0

### DISCUSSION THREAD

**Discussion by webmoskal** [seleniumbase/SeleniumBase#2470] [Q&A] [answered]
I'm facing an issue while trying to set up Selenium with Firefox on a Docker container derived from arm64v8/debian:bullseye-slim. I have installed Geckodriver version 0.34.0, but when I attempt to initiate a Firefox driver session using Selenium, I encounter a NoSuchDriverException error. Here are the details of my setup and the error message:
Environment:

    OS: Docker image based on arm64v8/debian:bullseye-slim
    Geckodriver Version: 0.34.0 , location /usr/local/bin/geckodriver
    Firefox: Mozilla Firefox 115.7.0esr location: /usr/bin/firefox
    Error Message: selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for Firefox using Selenium Manager.

cmd line: python -m pytest -vs --firefox --binary-location=/usr/bin/firefox

 would greatly appreciate any insights or suggestions on how to resolve this issue.

**Comment by mdmintz** [comment, marked as answer]
> "selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for Firefox using Selenium Manager."

That message is coming from raw Selenium (not SeleniumBase).

It looks like there's an existing ticket with info about it here: https://github.com/SeleniumHQ/selenium/issues/13434

If that error message came from inside SeleniumBase, then I could probably be of more help.
